docs/blog/2024-07-12-0.10-release.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Releasing 0.10
+description: TBD
+slug: littlehorse-0.10-release
+authors:
+- lh_council
+tags: [littlehorse, release]
+image: https://avatars.githubusercontent.com/u/140006313?s=400&u=7bf4c91d92dfe590ac71bb6b4821e1a81aa5b712&v=4
+hide_table_of_contents: false
+---
+
+The `0.10` release brings with it significant performance and reliability improvements. <!-- truncate -->
+
+## New Features
+
+### `lhctl` Binaries and Release Notes
+
+The `0.10.0` release comes with a new [Release Page](https://github.com/littlehorse-enterprises/littlehorse/releases), including artifacts with `lhctl` built for ARM, Intel, and Windows.
+
+### Reliability during Rebalances
+
+PR [#872](https://github.com/littlehorse-enterprises/littlehorse/pull/872) improves the reliability of LittleHorse during Kafka Streams rebalances. Previously, if a write request (eg. `rpc RunWf`) was received just before a rebalance, certain requests would "time out" from the client perspective and return a `DEADLINE_EXCEEDED` grpc error despite being properly accepted and processed by the server. This PR fixes that issue by redirecting the internal `rpc WaitForCommand` to the new destination for that command.
+
+### Rescue Failed Workflows
+
+PR [#883](https://github.com/littlehorse-enterprises/littlehorse/pull/883) allows users to restart failed `WfRun`'s via the `lhctl rescue` command. This is similar to allowing a user to execute mutating SQL queries via a CLI like `psql`.
+
+With this feature, a user can fix a buggy Task Worker implementation and then restart a failed `WfRun` and get it to execute the failed `TaskRun` again via:
+
+```
+lhctl rescue <wfRunId> <threadRunNumber>
+```
+
+### mTLS Principals
+
+Previously, only listeners of the type `OAUTH` supported `Principal`s. The `Principal` ID was determined by the OAuth Client ID or User Id. Release `0.10` introduces the ability to infer a `Principal` on an `MTLS` listener, where the `Principal` ID comes from the Common Name on the client certificate.
```

Comment:
Add a summary.  Something like "This enables users that standardize on SSL authentication to continue that scheme into Littlehorse."